framework/assets/yii.validation.js

```diff
@@ -36,7 +36,7 @@ yii.validation = (function ($) {
             }
         },
 
-        'boolean': function (value, messages, options) {
+        boolean: function (value, messages, options) {
```

Comment:
Got it. I'll return it as it was. I'll also add additional explaining comment about it in this case.